### ISSUE TITLE
getTokenSilently() returns with 5-part jwt (4 periods)

### ISSUE THREAD

**Issue by mhamrah** [auth0/auth0-spa-js#1008]
I'm getting a token with the `getTokenSilently()` operation, and I'm getting a string back that has four periods (see below). This seems like an invalid jwt which should have three parts to it. The code is straightforward:

```
const accessToken = await client.getTokenSilently();
    console.log(accessToken);
    return accessToken;
```
and the returned token:

`eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIiwiaXNzIjoiaHR0cHM6Ly9oYW1yYWguYXV0aDAuY29tLyJ9..8eNPbSQoVv0qKBKA.gv9d1TK5iJVDKgzWDuOBTDrCr39UkcKp4wvj-yZg-mS48kQpvg9mxmo05sCm2V5JKfaTDqKvcnOGp4o97aItDQNJxIfPhw5Id86XXicSljnewJ_bbV5DEG3q4VYkm9dapWyRuQW3Fx3i0KzTIJ0YClK7oaD8E6o8SavIFrNwOw2zhtXVX4a1xpIHJFYdNTmAUJ3Eo6EwHDiaW31BWw8BIKeKRN1FHnqO1IlLCgxi_qA3HM3rvLPJJuqBXaYBPZnMO2m07MlnkqpKHY7O-2GovktfY7UHTpyeBms6hPn8VXF061tmzA.pBt4_0LMVg78B7a9UzdkHg`

**Comment by frederikprijck** [comment]
Can you see the exact same token being returned by Auth0 when you look at the network tab? Can you please provide the corresponding HAR file for us to look into? Our SDK does not do anything with the Access Token but request it and store it.

**Comment by mhamrah** [comment]
Hi @frederikprijck the problem I realized this is an opaque id returned from auth0. I was following this documentation: https://auth0.com/docs/libraries/auth0-single-page-app-sdk#call-an-api expecting to get back a JWT, but I failed to put the audience of the api in the auth0 client initialization, so got the opaque id. When I added the audience, this returned a JWT. I left feedback on that documentation site to be more explicit about adding an audience parameter I case others get tripped up.
